### ISSUE TITLE
Batch normalization returns varaibles

### ISSUE THREAD

**Issue by StankoKuveljic** [smartcat-labs/facelyzr#27]
Batch normalization function should return beta and gamma variables. So in the per layer training (Stacked denoising autoencoder) optimizer can optimize these variables.
